packages/worker/src/api/routes/global/tests/auth.spec.ts

```diff
@@ -352,10 +313,43 @@ describe("/api/global/auth", () => {
 
     describe("GET /api/global/auth/:tenantId/oidc/callback", () => {
       it("logs in", async () => {
+        nock("http://someconfigurl").get("/").times(2).reply(200, {
+          issuer: "test",
+          authorization_endpoint: "http://localhost/auth",
+          token_endpoint: "http://localhost/token",
+          userinfo_endpoint: "http://localhost/userinfo",
+        })
+
+        const token = jwt.sign(
+          {
+            iss: "test",
+            sub: "sub",
+            aud: "clientId",
+            exp: Math.floor(Date.now() / 1000) + 60 * 60,
+            email: "oauth@example.com",
+          },
+          "secret"
+        )
+
+        nock("http://localhost").post("/token").reply(200, {
+          access_token: "access",
+          refresh_token: "refresh",
+          id_token: token,
+        })
+
+        nock("http://localhost").get("/userinfo?schema=openid").reply(200, {
+          sub: "sub",
+          email: "oauth@example.com",
+        })
+
         const configId = await generateOidcConfig()
         const preAuthRes = await config.api.configs.getOIDCConfig(configId)
-
         const res = await config.api.configs.OIDCCallback(configId, preAuthRes)
+        if (res.status > 399) {
```

Comment:
I usually prefer `res.status >= 400` - bit clearer that its truly only looking for error status!